src/cli.ts

```diff
@@ -312,16 +271,30 @@ add({
       throw new Error(`Unexpected existing types entry in package.json: ${json.types}`);
     }
 
-    const updated: Package = {
+    const updated: PackageJson = {
       ...json,
       types: proposedValue,
     };
 
-    try {
-      await fs.writeFile(filepath, stringify(updated));
-    } catch (e) {
-      throw new Error('Unable to write to package.json');
-    }
+    return await writeFile(filepath, updated);
+  },
+});
+
+add({
+  title: `Adding required entry-points to ${code('package.json')}`,
```

Comment:
For some reason this is not being called 🤔

Reply:
Wha?